pkg/services/vmoperator/vmopmachine_test.go

```diff
@@ -393,7 +393,7 @@ var _ = Describe("VirtualMachine tests", func() {
 			requeue, err = vmService.ReconcileNormal(ctx, supervisorMachineContext)
 			verifyOutput(supervisorMachineContext)
 
-			Expect(vmopVM.Spec.ReadinessProbe.TCPSocket.Port.IntValue()).To(Equal(defaultAPIBindPort))
+			Expect(vmopVM.Spec.ReadinessProbe.TCPSocket.Port.IntValue()).To(Equal(defaultAPIBindPort)) //nolint:staticcheck
```

Comment:
I'm really wondering why we only need this in the test code. It seems like our linter is not working for our production code (vmopmachine.go l.354)

Reply:
Okay no clue. Even if I disable all the excludes it doesn't find the issue in vmopmachine.go